pkg/apiserver/httputil/error.go

```diff
@@ -0,0 +1,31 @@
+// Copyright 2020 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package httputil
+
+import "github.com/gin-gonic/gin"
+
+// NewError example
+func NewError(ctx *gin.Context, status int, err error) {
```

Comment:
In latest master we have a better error framework now. You can remove it.

Errors can be passed by:

```
_ = c.Error(xxx)
return
```